cupy/creation/from_data.py

```diff
@@ -80,7 +80,7 @@ def ascontiguousarray(a, dtype=None):
 # TODO(okuta): Implement asmatrix
 
 
-def copy(a, order='C'):
+def copy(a, order='K'):
```

Comment:
Would you add a test to check `cupy.copy` works as same as `numpy.copy` on the default value of `order` option?